### ISSUE TITLE
Implement PSI job actions

### ISSUE THREAD

**Issue by sbliven** [SciCatProject/backend#1124]
Implement all actions needed for the PSI workflow in v4. This includes:

- [x] RabbitMQ (#1121)
- [x] Email with more complex templates
- [ ] Filters (eg different actions depending on the status)

**Comment by despadam** [comment]
Filters to be implemented later for OpenEM when ETH is also onboarded.

**Comment by sbliven** [comment]
Currently blocked by #1154

**Comment by despadam** [comment]
Separate issue for Filters action: https://github.com/SciCatProject/scicat-backend-next/issues/1658 (not needed for the MVP)

**Comment by despadam** [comment]
Separate issue for job action testing: https://github.com/SciCatProject/scicat-backend-next/issues/1435
